doc/release-notes.md

```diff
@@ -74,6 +74,13 @@ Due to a backward-incompatible change in the wallet database, wallets created
 with version 0.16.0 will be rejected by previous versions. Also, version 0.16.0
 will only create hierarchical deterministic (HD) wallets.
 
+Custom wallet directories
+---------------------
+The ability to specify a directory other than the default data directory in which to store
+wallets has been added. An existing directory can be specified using the `-walletdir=<dir>`
+argument. The parameter defaults to using the data directory location if not included.
+Wallets loaded via `-wallet` arguments are relative to this wallet directory.
```

Comment:
I'm not sure, but perhaps we should add a warning that `-walletdir` must refer to a reliable directory in the filesystem. For example, I think we want to discourage people using ephemeral, removable storage for their `-walletdir` since if the directory becomes unavailable during operation, funds may be lost.